### PR TITLE
Promo page

### PR DESCRIPTION
**Pull request by avkozlov4**

### CODE REVIEW

File: src/components/DevDoBlock/devdo-block.scss [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+  }
+
+  &-text-box {
+    flex: 1 0 0;
+
+    .title-box {
+      display: flex;
+      justify-content: flex-end;
+
+      h3 {
```
Comment (marina-marina) [inline src/components/DevDoBlock/devdo-block.scss]: plz use classes or `>` for all the tags

File: src/components/DevDoBlock/devdo-block.scss [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+        font-weight: 400;
+        line-height: 45px;
+        letter-spacing: 2px;
+        -webkit-font-smoothing: antialiased;
+        margin-top: 5px;
+        margin-left: auto;
+      }
+    }
+
+    p {
```
Comment (marina-marina) [inline src/components/DevDoBlock/devdo-block.scss]: same

File: src/components/DevDoBlock/devdo-block.scss [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+
+    &:nth-child(2n+1) .devdo-block-video {
+      margin-right: 0;
+    }
+
+    .anchor {
+      order: 1;
+    }
+
+    .title-box h3 {
```
Comment (marina-marina) [inline src/components/DevDoBlock/devdo-block.scss]: same

File: src/components/DevDoBlock/devdo-block.scss [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+    .buy-tickets {
+      margin-bottom: 25px;
+    }
+  }
+}
+
+@media screen and (max-width: 600px) {
+  .devdo-block {
+
+    .title-box h3 {
```
Comment (marina-marina) [inline src/components/DevDoBlock/devdo-block.scss]: same here

File: src/layouts/common.scss [hunk 5]
```diff
@@ -81,6 +80,12 @@ html, body, #___gatsby {
...
   position: relative;
   box-sizing: border-box;
+  background-color: $blue;
+
+  &.bg-grey {
+    background-color: $grey-bg;
+  }
+
   &-body {
     h1, p {
```
Comment (marina-marina) [inline src/layouts/common.scss:90]: and here

File: src/pages/promo.scss [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+    @include block-header;
+  }
+
+  .devdo-list {
+    display: grid;
+    grid-row-gap: 105px;
+    margin-top: 45px;
+  }
+
+  .devdo-block-text a {
```
Comment (marina-marina) [inline src/pages/promo.scss]: and here

File: src/layouts/common.scss (hunk 5 above)
Comment (avkozlov4) [reply src/layouts/common.scss:90]: It's global styles for `h1` and `p`, i suppose that some code uses a color definition based on common CSS

File: src/pages/promo.scss (hunk 6 above)
Comment (avkozlov4) [reply src/pages/promo.scss]: Description for this block presume any level of nesting ( `>` may not work), and `className` may not be defined due to JSON source. So `.devdo-block-text a` is more reliable solution in this case

**Comment by tyv** [comment]
@marina-marina I fixed ugly titles padding on speakers page. 
recheck plz everything (there is +one page)
fix if you think necessary and merge/deploy, plz

**Pull request merged**